packages/aggregate-api/src/types.ts

```diff
@@ -0,0 +1,77 @@
+import type {
+  HandlerExecutionError,
+  Signer,
+  InboundCodec,
+} from '@ucanto/interface'
+
+import type { Offer } from '@web3-storage/aggregate-client/types'
+export * from '@web3-storage/aggregate-client/types'
+
+export * from '@web3-storage/capabilities/types'
+export * as UcantoInterface from '@ucanto/interface'
+
+export interface AggregateServiceContext {
+  aggregateStore: AggregateStore
+  offerStore: OfferStore
+}
+
+export interface OfferServiceContext {
+  arrangedOfferStore: ArrangedOfferStore
+}
+
+export interface ServiceContext
+  extends AggregateServiceContext,
+    OfferServiceContext {}
+
+export interface ArrangedOfferStore {
+  get: (commitmentProof: string) => Promise<string | undefined>
```

Comment:
We should look into this as a major change of APIs. We have all other (access-api, upload-api, ...) as we have here. I would prefer for now to be uniform across the board instead of needing to have different mental models